features/insured/step_definitions/individual_steps.rb

```diff
@@ -54,3 +54,27 @@
     click_link "CONTINUE"
   end
 end
+
+When(/individual has a home and mailing address/) do
+  addresses = Person.first.addresses
+  addresses.last.destroy
+  addresses << FactoryBot.build(:address, :mailing_kind)
+  addresses.last.save!
+end
+
+When(/individual removes mailing address/) do
+  find(IvlPersonalInformation.remove_mailing_address_btn).click
+end
+
+When(/individual edits home address/) do
+  fill_in IvlPersonalInformation.address_line_one, :with => "123 New St"
+end
+
+When(/individual saves personal information changes/) do
+  find_all('.btn-primary').first.click
```

Comment:
this is very brittle - let's add a data-cuke attribute to target the button that needs to be clicked instead of hoping it's always the first result for `.btn-primary`